02-wrangling.Rmd

```diff
@@ -328,35 +519,39 @@ through and find where `output_1` and `output_2` are used in each subsequent lin
 
 Another option for doing this would be to *compose* the functions:
 
-```
+```{r eval = F}
 output <- select(filter(mutate(data, new_col = old_col * 2), other_col > 5), new_col)
 ```
 
 Code like this can also be difficult to understand. Functions compose (reading from left to right) in the *opposite order* in which
 they are computed by R (above, `mutate` happens first, then `filter`, then `select`). It is also just a really long line of code
 to read in one go. 
 
-The *pipe operator* `%>%` solves this problem, resulting in cleaner and easier-to-follow code. The below accomplishes the same thing as the previous two code blocks:
+The *pipe operator* `|>` solves this problem, resulting in cleaner and easier-to-follow code. The below accomplishes the same thing as the previous two code blocks:
```

Comment:
We could write "built-in" instead of base R? Not sure which is more understandable to new learners?
```suggestion
The base R *pipe operator* `|>` solves this problem, resulting in cleaner and easier-to-follow code. The below accomplishes the same thing as the previous two code blocks:
```